package.json

```diff
@@ -75,7 +75,8 @@
     "coverage": "npm run test -- --coverage --watchAll=false",
     "storybook": "start-storybook -p 9009 -s public",
     "build-storybook": "build-storybook -s public",
-    "codegen": "graphql-codegen --config codegen.js"
+    "codegen": "graphql-codegen --config codegen.js",
+    "deploy-prod:do-not-use-directly": "npm run build:prod && BUCKET=evergreen.spruce npm run deploy"
```

Comment:
I didn't catch this before, but S3 buckets that we own shouldn't be part of the build system in an open source project. Can you make the staging and prod buckets params in the .cmdrc?

Reply:
in the process of chaning, but why does exposing bucket name matter if the bucket requires authorization to access?

Reply:
zakhar on build team does not consider bucket names a secret. given this in addition to the level of security surrounding our buckets, exposing bucket names is not compromising.

Reply:
It's not a security concern, settings specific to our deployment belong in a configuration file rather than hardcoded in the repo

Reply:
it's not super straight forward to pull off from what i can tell. if it's important to you can you create a ticket please?

Reply:
Yep created EVG-8202

Reply:
thank you sir!